source/opt/inline_pass.cpp

```diff
@@ -202,14 +245,17 @@ void InlinePass::GenInlineCode(
   // Clone and map callee code. Copy caller block code to beginning of
   // first block and end of last block.
   bool prevInstWasReturn = false;
+  uint32_t headerId = 0;
```

Comment:
headerId is too generic a name.  Recommend singleTripLoopHeaderId for this, and singleTripLoopContinueId for the next var.